PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -33,14 +33,13 @@
         binaries.append((f, 'lib/gdk-pixbuf-2.0/2.10.0/loaders'))
     
     # Create an updated version of the loader cache
-    cachedata = subprocess.check_output('gdk-pixbuf-query-loaders')
+    cachedata = subprocess.check_output('gdk-pixbuf-query-loaders', universal_newlines=True)
```

Comment:
Why is universal_newlines necessary here? The content is written to the new file unchanged for all platforms except OSX. And OSX already uses `\n` AFAIK. (If not, newlines should be converted for `is_dawrin` only.)